app/lib/common/widget/ChatOverview.dart

```diff
@@ -69,32 +69,12 @@ class ChatListItem extends StatelessWidget {
           ),
         );
       },
-      leading: FutureBuilder<Uint8List>(
-        future: room.avatar().then((fb) => fb.asTypedList()),
-        // a previously-obtained Future<String> or null
-        builder: (BuildContext context, AsyncSnapshot<List<int>> snapshot) {
-          if (snapshot.connectionState == ConnectionState.waiting) {
-            return SizedBox(
-              height: 20,
-              width: 20,
-              child: CircularProgressIndicator(color: AppColors.primaryColor),
-            );
-          } else {
-            if (snapshot.hasData) {
-              return CircleAvatar(
-                radius: 25,
-                backgroundImage:
-                    MemoryImage(Uint8List.fromList(snapshot.requireData)),
-              );
-            } else {
-              return CircleAvatar(
-                radius: 25,
-                backgroundColor: Colors.grey[700],
-                child: SvgPicture.asset('assets/images/people.svg'),
-              );
-            }
-          }
-        },
+      leading: CustomAvatar(
```

Comment:
is this in anticipation? because the custom Avatar isn't in here and isn't merged yet (and the `.avatar()` interface isn't implemented yet)

Reply:
This is outdated, I merged the PR earlier.